sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyAsyncClient.java

```diff
@@ -81,38 +61,19 @@
  * @see KeyClientBuilder
  * @see PagedFlux
  */
-@ServiceClient(builder = KeyClientBuilder.class, isAsync = true, serviceInterfaces = KeyService.class)
+@ServiceClient(builder = KeyClientBuilder.class, isAsync = true)
```

Comment:
We should continue providing `serviceInterfaces = <the interface RestProxy uses>`

Reply:
Can we use service interface that's in impl package here ?

Reply:
We don't do that here https://github.com/Azure/azure-sdk-for-java/blob/639bff34391d302cac1e351eaac3197c69e415b7/sdk/anomalydetector/azure-ai-anomalydetector/src/main/java/com/azure/ai/anomalydetector/AnomalyDetectorClient.java#L35